target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz3490.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz3490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 3490";
+
+	memory@0 {
```

Comment:
This part is common to all devices. I would consider moving this to vr9_avm_fritzxx90.dtsi.

Reply:
Yes, I will move that.